azkaban-common/src/main/java/azkaban/executor/ExecutorLoader.java

```diff
@@ -42,6 +42,9 @@ ExecutableFlow fetchExecutableFlow(int execId)
   Map<Integer, Pair<ExecutionReference, ExecutableFlow>> fetchUnfinishedFlows()
       throws ExecutorManagerException;
 
+  Map<Integer, Pair<ExecutionReference, ExecutableFlow>> fetchUnfinishedExecutions()
```

Comment:
The method names `fetchUnfinishedExecutions` and `fetchUnfinishedFlows` are too similar to each other. Would it be better to use the name `fetchUnfinishedFlowsMetadata` since you are only fetching metadata info about the flow?